libraries/lib-numeric-formats/NumericConverter.cpp

```diff
@@ -176,12 +176,23 @@ struct BuiltinFormatString
          { return a.name == b.name; }
 };
 
+NumericField::NumericField(
+   bool _frac, int _base, int _range, bool _zeropad) noexcept
+    : frac { _frac }
+    , base { _base }
+    , range { _range }
+    , zeropad { _zeropad }
```

Comment:
This naming convention for ctor arguments isn't necessary.  They can just be the same as the members.